src/main/java/pl/edu/mimuw/Bank.java

```diff
@@ -1,25 +1,96 @@
 package pl.edu.mimuw;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class Bank {
 
+  private List<BankClient> allClients;
+  private String name;
+  private String slogan;
+  private int numberOfClients;
+
+  public Bank(String name, String slogan) {
+    this.name = name;
+    this.slogan = slogan;
+    this.allClients = new ArrayList<>();
+    numberOfClients = 0;
+  }
+
   @Override
   public String toString() {
-    throw new IllegalStateException("TODO: get information about bank and all its clients (use StringBuilder)");
+    StringBuilder sb = new StringBuilder();
+    sb.append(name);
+    sb.append("\n");
+    sb.append(slogan);
+    sb.append("\n");
+    if (numberOfClients == 0) {
+      return sb.toString();
+    } else {
+      sb.append("KLIENCI:" + "\n");
+      for (int i = 0; i < numberOfClients; i++) {
+        sb.append(allClients.get(i).toString());
+        sb.append("\n");
+      }
+      return sb.toString();
+    }
   }
 
+  /**
+   * @return Wazne - zwraca kopie listy wszystkich klientow wypelniona kopiami tychze
+   */
   public List<BankClient> getAllClients() {
-    throw new IllegalStateException("TODO: remember not to return internal state list as it can be modified");
+    if (allClients == null) {
```

Comment:
Ty zarządzasz tym polem i chyba zawsz eni jest nullem, więc niepotrzebne to :wink: